mne/source_space.py

```diff
@@ -1551,7 +1566,7 @@ def setup_volume_source_space(subject, fname=None, pos=5.0, mri=None,
     # Save it
     sp.update(dict(nearest=None, dist=None, use_tris=None, patch_inds=None,
                    dist_limit=None, pinfo=None, ntri=0, nearest_dist=None,
-                   nuse_tri=0, tris=None))
+                   nuse_tri=0, tris=None, subject_his_id=subject))
```

Comment:
Now it is also setting ``subject_his_id`` to None when reading a source space if no tag is found. The tests would fail otherwise because the key would be missing.